doc/ref_model/chapters/chapter05.md

```diff
@@ -88,8 +88,8 @@ The following sections detail the NFVI SW profile features per type of virtual r
 
 | .conf | Feature | Type  | Description |
 |------------------|----------------|----------------|------------------------------------------------------------------------------------------------|
-| nfvi.net.cfg.001 | vNIC interface | IO virtualisation | e.g. virtio1.1, i40evf (Intel driver for VF SR-IOV). |
-| nfvi.net.cfg.002 | Overlay protocol | Protocols | The overlay network encapsulation protocol needs to enable ECMP in the underlay to take advantage of the scale-out features of the network fabric. |
+| nfvi.net.cfg.001 | vNIC interface | IO virtualisation | e.g. virtio1.1, i40evf (Intel driver for VF SR-IOV). <br> **Note:** This feature is not applicable to containers or Kubernetes based infrastructures. |
```

Comment:
There is another PR proposing to remove vendor specific drivers https://github.com/cntt-n/CNTT/pull/653. does it make sense to wait until that PR is concluded? Alternatively, we can bring in the changes proposed in https://github.com/cntt-n/CNTT/pull/653 into here.

Reply:
I've removed the related changes.